selfie.dev/src/pages/py/get-started.mdx

```diff
@@ -1,47 +1,31 @@
 import { DocsImage } from "@/components/DocsImage";
 
 export const title = "Selfie Python Snapshot Testing | Get started";
-export const description = "Zero-config inline and disk snapshots for Python. Add a single dependency and you're all set.";
+export const description =
+  "Zero-config inline and disk snapshots for Python. Add a single dependency and you're all set.";
 export const imageUrl = "https://selfie.dev/get-started.webp";
 
 <DocsImage imgAbsoluteUrl={imageUrl} />
 
 TODO: adapt this to Python
 
-To start snapshot testing in Java or any JVM language, all you need is to add a [single dependency](https://central.sonatype.com/artifact/com.diffplug.selfie/selfie-runner-junit5).
+To start snapshot testing in Python, all you need is to add a [single dependency](https://central.sonatype.com/artifact/com.diffplug.selfie/selfie-runner-junit5).
```

Comment:
Is this where I link pytest-runner and selfie-lib? or just the runner?

Reply:
Just the runner.